.github/workflows/test.yml

```diff
@@ -7,9 +7,18 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       # - uses: actions/checkout@v2
-      - uses: shenxianpeng/cpp-linter-action@master
+      # - uses: shenxianpeng/cpp-linter-action@master
+      - uses: 2bndy5/cpp-linter-action@output-exit-code
```

Comment:
Do you need to revert this change before merging?

Reply:
Yes I do. Good catch.